components/n-ui/tracking/third-party/chartbeat.js

```diff
@@ -6,7 +6,7 @@ Eventually this will be replaced by FT's in-house "lantern" application, utilisi
 */
 
 import {cookieStore} from 'n-ui-foundations';
-import {loadScript} from '../../../../browser/bootstrap/js/utils';
+import {loadScript} from '../../../../browser/js/utils';
```

Comment:
Yep, this is actually already in master but for some reason has been added in this PR. A rebase gone awry I think.